packages/app/src/app/pages/Sandbox/Editor/utils/get-vscode-theme.js

```diff
@@ -97,10 +97,19 @@ const findTheme = async (themeName, customTheme) => {
   };
 };
 
+const classnames = {
+  dark: 'vs-dark',
+  light: 'vs',
+  hc: 'hc-black',
+};
+
 export default async function getTheme(themeName, customTheme) {
   const foundTheme = await findTheme(themeName, customTheme);
   // Explicitly check for dark as that is the default
   const isLight = foundTheme.type !== 'dark' && foundTheme.type !== 'hc';
+  document.body.classList.remove('vs');
+  document.body.classList.remove('vs-dark');
```

Comment:
Whoops, we forgot to add `hc-black` here!

```suggestion
  document.body.classList.remove('vs-dark');
  document.body.classList.remove('hc-black');
```